test/remote/too_many_login_checks.js

```diff
@@ -158,6 +167,105 @@ test(
   }
 )
 
+test(
+  '/checkAuthenticated with same action',
```

Comment:
@larissagaulia we should move these tests to their own file, `/checkAuthenticated` is not part of `test/remote/too_many_login_checks.js`